task/validator.go

```diff
@@ -254,11 +254,17 @@ func validateBucket(ctx context.Context, script string, preAuth query.PreAuthori
 
 	spec, err := flux.Compile(ctx, script, time.Now())
 	if err != nil {
-		return err
+		return platform.NewError(
+			platform.WithErrorErr(err),
+			platform.WithErrorMsg("Failed to parse flux script."),
```

Comment:
Maybe this should be `compile` rather than `parse`?

Reply:
I like compile here!